.ruby-version

```diff
@@ -1 +1 @@
-ruby-2.6
+2.6.0
```

Comment:
What was the reason for this change? 🙂

Reply:
This was changed by rbenv, it refers to `ruby-2.6` as just `2.6.0`.

I can remove that file if you think it will cause a problem. TravisCI didn't seem to have an issue though

Reply:
ahh I see, can we remove the `0` so it's just `2.6` that should allow it to work with any 2.6.x version of ruby.

Reply:
Ill make it happen